docs/tutorials/sending-and-receiving-messages-with-mqtt.md

```diff
@@ -49,9 +49,15 @@ working:
 Now we are able to send and receive messages through the broker (by default
 mosquitto uses port 1883). 
 
-This tutorial uses python scripts so we need to install one python library:
+This tutorial uses python scripts so we need to install the python library paho-mqtt.
+You need 'pip3' to install this module, so if you have not already done so, you will
+need to install 'pip3':
 
-    sudo easy_install paho-mqtt
+  sudo apt-get install python3-pip
+
+ Now you can install paho-mqtt:
+ 
+  sudo pip3 install paho-mqtt
 
 All scripts were tested successully on a EV3 running the latest ev3dev version
 (as of 12 May 2016) and also on a Raspberry Pi 3 with a BrickPi running the same
```

Comment:
2016-12-21 release, no updates at all, first script works fine just changed the shebang to python3:

```
Linux ev3dev 4.4.32-17-ev3dev-ev3 #1 PREEMPT Wed Nov 16 11:12:06 CST 2016 armv5tejl GNU/Linux


sudo apt-get install python3-pip mosquitto
systemctl status mosquitto

● mosquitto.service - LSB: mosquitto MQTT v3.1 message broker
   Loaded: loaded (/etc/init.d/mosquitto)
   Active: active (running) since Fri 2016-12-23 09:22:58 UTC; 4min 0s ago
   CGroup: /system.slice/mosquitto.service
           └─1341 /usr/sbin/mosquitto -c /etc/mosquitto/mosquitto.conf

sudo pip3 install paho-mqtt

pub01.py:

#!/usr/bin/env python3

import paho.mqtt.client as mqtt

# This is the Publisher

client = mqtt.Client()
client.connect("localhost",1883,60)
client.publish("topic/test", "Hello world!");
client.disconnect();

OK
```